### DISCUSSION TITLE
gqlgen command gets stuck after upgrading from 0.13 to 0.16

### DISCUSSION THREAD

**Discussion by dimastark** [99designs/gqlgen#1886] [Q&A]
### What happened?

I updated the library version from 13 to 16, after which the code generation started to hang.
I tried to run with the verbose flag, but the logs are empty.

### What did you expect?

Code generation works as before

### versions
 - `gqlgen version`? 
 
0.16.0

 - `go version`? 
 
go version go1.17.6 linux/amd64

 - dep or go modules? 
 
go modules

**Comment by frederikhors** [comment]
Please create a Github repo to reproduce the issue.
